packages/terra-search-field/src/SearchField.module.scss

```diff
@@ -47,6 +47,11 @@
       right: 0;
       top: 0;
 
+      &.error-clear {
+        outline: var(--terra-search-field-focus-outline, 2px solid #e50000);
+        outline-offset: var(--terra-search-field-focus-outline-offset, 1px);
+      }
```

Comment:
Why do we need error outline on clear button. are these styles provided UX team ?